### ISSUE TITLE
Need minor updates to getting started documentation/config.

### ISSUE THREAD

**Issue by patricknelson** [r-pop/findthemasks#210]
@csakoda 

From PR #196, originally called out in https://github.com/r-pop/findthemasks/issues/185#issuecomment-604146287

Can you make a quick fix for this? Following instructions in `getting_started.md` results in a bare directory listing. Is it possible to refine this such that it's still cross platform compatible but still has a fairly simple setup (like what you already created) but uses the `public/` directory instead? Potentially we could just create a script in `package.json` that launches `http-server` with the extra config necessary for this .

![image](https://user-images.githubusercontent.com/4269377/77595991-fe230580-6eb7-11ea-944e-f8c5c3cd27cb.png)

**Comment by csakoda** [comment]
It looks like you just need to run http-server one dir down.  Let me see if I can remove the need for that `cd ..`

<img width="301" alt="image" src="https://user-images.githubusercontent.com/4173737/77596312-ea2bd380-6eb8-11ea-82fa-dfdfdb035279.png">

**Comment by patricknelson** [comment]
Ok, so in this case there are two `node_module`'s right? One for firebase, another for the local dev, is that required for `http-server` or can `http-server` run entirely independently without the `npm i` in `functions/`?

**Comment by csakoda** [comment]
`npm install -g http-server` puts it in the global npm cache (we're bumping up against the limits of my npm knowledge though)

**Comment by patricknelson** [comment]
Think we should update the docs then too; earlier today @cjimmy noticed this as well.

**Comment by csakoda** [comment]
What update do you have in mind?  As best as I can tell the only problem with the steps if that if you don't `cd ..` you'll get confused as to why the site isnt loading (by default, http-server expects to find a `public/` dir in the path you start it).

**Comment by patricknelson** [comment]
Yeah basically that; ideally it'd be very "hand holdy" so new users, particularly since it's not immediately obvious to those (like myself) how `http-server` works. I think that's why we were tripped up.

**Comment by crayne** [comment]
Do I need to install firebase explicitly to run findthemasks locally?  If so, I'm sure from getting_started how to do this.

**Comment by csakoda** [comment]
@crayne No, you shouldn't need firebase to do dev on the frontend site.
